tests/unit/cluster/announced-endpoints.tcl

```diff
@@ -8,13 +8,16 @@ start_cluster 2 2 {tags {external:skip cluster}} {
         }
         set count [expr [llength $::servers] + 1]
         set used_port [find_available_port $baseport $count]
+        R 0 CLUSTER SLOTS
+        R 1 CLUSTER SLOTS
```

Comment:
Is this to trigger the debug assertions? Should we document that if it's true, otherwise it's not clear what these dangling commands are for.

Reply:
Yes, I forgot to add the comment. Will be adding it next.